client/packages/invoices/src/StockOut/hooks/usePackSizeController.test.tsx

```diff
@@ -37,6 +38,7 @@ const testLine = ({
 }: TestLineParams): DraftStockOutLine =>
   createDraftStockOutLine({
     invoiceId: '',
+    invoiceStatus: InvoiceNodeStatus.New,
```

Comment:
Didn't look at these tests but they do seem to pass with `New` so I left it as New for now